### PR TITLE
Comments CSS Challenge

### PR DESCRIPTION
**Pull request by eschwartz**
Using this as the weekend challenge for Gaiman bridge week. Will not be graded, but I plan to do a "live solve" on Monday (wish me luck 🍀 😬 )

Adapted from Frontend Masters:
https://www.frontendmentor.io/challenges/interactive-comments-section-iG1RugEG9

Files changed in this PR are changes vs. the original:
- Removed any JS requirements, including rendering out the `data.json` file
- Removed mobile design
- Simplified / clarified the instructions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,20 +3,278 @@
...
+          <article>
+            Impressive! Though it seems the drag feature could be 
+            improved. But overall it looks incredible. 
+            You've nailed the design and the responsiveness at 
+            various breakpoints works really well.
+          </article>
+        </section>
+      </li>
+      
+      <!-- A Single Comment (top level) -->
```
Comment (emkerber) [inline index.html]: Looks like this is currently structured to have three parent comments and one child comment - should this third comment `<li>` be nested within the second/inner `<ul>` for replies to the parent comment?

File: index.html [hunk 2]
```diff
@@ -3,20 +3,278 @@
...
+          <button>
+            <img src="./images/icon-plus.svg" alt="upvote" />
+          </button>
+          
+          <!-- Current points -->
+          <div class="points">12</div>
+          
+          <!-- Downvote -->
+          <button>
+            <img src="./images/icon-minus.svg" alt="upvote" />
```
Comment (emkerber) [inline index.html]: alt="downvote"

**Comment by eschwartz** [comment]
@emkerber thanks for reviewing, but I just realized that I pushed up my lecture code (solution), instead of the original assignment.

I'm going to move the completed lecture code to a `solution` branch, and remove the solution commits from `main`

**Comment by eschwartz** [comment]
k, I totally borked this PR, it won't let me reopen it... I'll make a new one

**Pull request closed without merging**